src/style.css

```diff
@@ -1,3 +1,65 @@
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+  background-color: lightgray;
+}
+
+header {
+  display: flex;
+  padding: 0 50px;
+  background-color: white;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  margin: 10px 0;
+}
+
+.nav {
+  display: flex;
+  align-items: center;
+}
+
+.nav__list {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  cursor: pointer;
+  justify-content: flex-end;
+
+}
+
+.nav__item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.nav__link {
+  line-height: 60px;
+  padding: 23px 0;
+  font-size: 12px;
+  color: black;
+  text-decoration: none;
+  text-transform: uppercase;
+}
+
+.is-active,
+.nav__link:hover {
+  line-height: 60px;
+  color: #00acdc;
```

Comment:
Consider using variables for repeating colors

Reply:
done, should i execute pull request again?